parsers/to-react-native/README.md

```diff
@@ -0,0 +1,150 @@
+# To React Native
+
+## Description
+
+Transform design tokens to a JavaScript `theme` object which can be used in React Native.
+
+## Usage
+
+### Options
+
+| Parameter          | Required | Type                                                                       | Default     | Description                                                                                                           |
+| ------------------ | -------- | -------------------------------------------------------------------------- | ----------- | --------------------------------------------------------------------------------------------------------------------- |
+| `colorFormat`      | optional | `rgb`, `prgb`, `hex`, `hex6`, `hex3`, `hex4`, `hex8`, `name`, `hsl`, `hsv` | `rgb`       | The format of all colors (gradients, borders, textStyles etc.)                                                        |
+| `objectName`       | optional | `string`                                                                   | `'theme'`   | The name of the JS object containing the theme,                                                                       |
+| `assetsFolderPath` | optional | `string`                                                                   | `undefined` | The relative location of the folder to import the assets from, if not provided, the assets will be referenced by URL. |
+| `prettierConfig`   | optional | [`prettier.Config`](https://prettier.io/docs/en/options.html)              | `undefined` | Formatting options of the file.                                                                                       |
+
```

Comment:
Please set `## Types` as the third section of the README (i.e. https://github.com/Specifyapp/parsers/blob/master/parsers/to-scss-mixin-text-style/README.md#types).

This section must include the types of data that is accepted as input by the parser and the type of data that is returned by the parser. This helps our user understand which parser can be chained with each other.

End result 👇

---

## Types

ℹ️ **Please be aware that, depending on the order you use parsers, their input and output types have to match.**

### Input

Array of object with at least name, value and type

```ts
Array<{name: string, value: any, type: string}>
```

### Output

```ts
string
```